mavis/validate/call.py

```diff
@@ -350,7 +350,7 @@ def characterize_repeat_region(event, reference_genome):
                 break
             repeat_count += 1
             rightmost -= len(expected_sequence)
-        return repeat_count
+        return repeat_count, expected_sequence
```

Comment:
can you add a description of the return to the docstring here?

Reply:
Added a description to docstring